options/options.c

```diff
@@ -140,8 +140,10 @@ static const m_option_t mp_vo_opt_list[] = {
     {"native-keyrepeat", OPT_BOOL(native_keyrepeat)},
     {"panscan", OPT_FLOAT(panscan), M_RANGE(0.0, 1.0)},
     {"video-zoom", OPT_FLOAT(zoom), M_RANGE(-20.0, 20.0)},
-    {"video-pan-x", OPT_FLOAT(pan_x), M_RANGE(-3.0, 3.0)},
-    {"video-pan-y", OPT_FLOAT(pan_y), M_RANGE(-3.0, 3.0)},
+    {"video-pan-x", OPT_FLOAT(pan_x)},
+    {"video-pan-y", OPT_FLOAT(pan_y)},
+    {"output-pan-x", OPT_FLOAT(outpan_x)},
+    {"output-pan-y", OPT_FLOAT(outpan_y)},
```

Comment:
not sure how I feel about `output-pan`, but I have not better idea.

Reply:
Imo it makes sense, since it's relative to the output dimensions whereas video-pan is relative to the video dimensions.